string vocal_command
---
bool output_signal
